pkg/api/server/v1alpha2/log/log_test.go

```diff
@@ -112,13 +112,13 @@ func TestParseName(t *testing.T) {
 }
 
 func TestToStorage(t *testing.T) {
-	log := &v1alpha2.Log{
+	log := &v1alpha3.Log{
```

Comment:
should we not have at least one v1alpha2 test case to validate that behavior?

Reply:
The tests won't work because we no longer have v1alpha2, to simulate, we have to retain the v1alpha2 types as well.